kivy/core/camera/camera_opencv.py

```diff
@@ -106,7 +106,7 @@ def init_camera(self):
 
         elif self.opencvMajorVersion in (2, 3, 4):
             # create the device
-            self._device = cv2.VideoCapture(self._index)
+            self._device = cv2.VideoCapture(self._index, cv2.CAP_DSHOW)
```

Comment:
`cv2.CAP_DSHOW` is Windows only, and also on Windows the user may want to start the capture with another API.

Maybe you can introduce a logic that allows the user to select a specific API?

Reply:
Well that took longer than I anticipated, but it's done.  Let me know if you have any other feedback.  I don't have a lot of experience with public projects on GitHub so suggestions are welcome.